lib/stale.js

```diff
@@ -12,25 +12,34 @@ module.exports = class Stale {
 
     await this.ensureStaleLabelExists();
 
-    this.getIssues().then(paginate(this.github, data => {
-      data.items.forEach(issue => {
-        if (this.isStale(issue)) {
-          if (this.hasStaleLabel(issue)) {
-            return this.close(issue);
-          } else {
-            return this.mark(issue);
-          }
-        }
-      });
+    this.getStaleIssues().then(paginate(this.github, data => {
+      data.items.forEach(issue => this.mark(issue));
     }));
+
+    this.getClosableIssues().then(paginate(this.github, data => {
+      data.items.forEach(issue => this.close(issue));
+    }));
+  }
+
+  getStaleIssues() {
+    const labels = [this.config.staleLabel].concat(this.config.exemptLabels);
+    const query = labels.map(label => `-label:${label}`).join(' ');
```

Comment:
I'm trying to debug why the stale issues are not being marked as stale and it seems it is because we have some labels with spaces and those spaces don't work really well in the search.

If I get the search query for example and put in the github search I get an empty set:

```
repo:rails/rails is:open updated:<2017-02-04T00:29:52 -label:stale -label:pinned -label:security -label:With reproduction steps -label:attached PR -label:regression -label:release blocker is:issue
```

But if I remove all labels with space on the name I get some issue that would match the query. Does `"` works for labels?